templates/core/build.html

```diff
@@ -5,151 +5,180 @@
-      <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
-      <hr class="u-sv1"/>
-      <p class="p-card__content">Core 16</p>
-    </button>
-    <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3 raspberry-pi4 intel-nuc qualcomm-dragonboard">
-      <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
-      <hr class="u-sv1"/>
-      <p class="p-card__content">Core 18</p>
-    </button>
-    <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3">
-      <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
-      <hr class="u-sv1"/>
-      <p class="p-card__content">Classic 16.04</p>
-    </button>
-    <button class="col-2 p-card js-selection" data-supports="raspberry-pi2 raspberry-pi3 raspberry-pi4">
-      <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
-      <hr class="u-sv1"/>
-      <p class="p-card__content">Classic 18.04</p>
-    </button>
-    <button class="col-2 p-card js-selection" data-supports="raspberry-pi3 raspberry-pi4">
-      <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg" alt="Ubuntu logo">
-      <hr class="u-sv1"/>
-      <p class="p-card__content">Classic 64-bit 18.04</p>
-    </button>
+    {% endif %}
   </div>
 </section>
 
-<section class="p-strip is-shallow">
-  <div class="row">
-    <div class="col-8">
-      <h2>What snaps do you want preinstalled?</h2>
-      <p>Select the applications that will be pre-installed on the device. For Ubuntu Core this will be snaps, for Ubuntu Classic it will be snaps or packages.</p>
+<div {% if not openid %}style="opacity: .2; pointer-events: none;"{% endif %}>
+  <section class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-8">
+        <h2>1 What board are you using?</h2>
+      </div>
     </div>
-  </div>
-  <div class="row p-divider">
-    <div class="col-6 p-divider__block">
-      <form class="p-search-box js-snap-search">
-        <input type="search" class="p-search-box__input" name="search" placeholder="Search for snaps" required="">
-        <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close"></i></button>
-        <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
-      </form>
-      <div class="js-snap-results"></div>
+    <div class="row js-boards">
+      <button class="col-2 p-card js-selection">
+        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/8542c958-Raspberry+Pi+2.png" alt="A photo the the Raspberry Pi 2">
+        <hr class="u-sv1">
+        <p class="p-card__content">Raspberry Pi2</p>
```

Comment:
“One What board are you using? A photo the the Raspberry Pi two. Raspberry Pi two” 😕

Here, “[the icon is supplemental to a text label conveying the same meaning. In those cases, the alt attribute must be present but must be empty.](https://html.spec.whatwg.org/multipage/images.html#a-short-phrase-or-label-with-an-alternative-graphical-representation:-icons,-logos)”

The same for all the other boards, and all the Ubuntu versions.

Reply:
I agree, done